src/EA.Iws.RequestHandlers/NotificationMovements/BulkReceiptRecovery/ReceiptRecoveryShipmentMustBePrenotifiedRule.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace EA.Iws.RequestHandlers.NotificationMovements.BulkReceiptRecovery
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Core.Movement;
+    using Core.Movement.BulkReceiptRecovery;
+    using Core.Rules;
+    using Domain.Movement;
+
+    public class ReceiptRecoveryShipmentMustBePrenotifiedRule : IReceiptRecoveryContentRule
+    {
+        private readonly IMovementRepository repo;
+
+        public ReceiptRecoveryShipmentMustBePrenotifiedRule(IMovementRepository repo)
+        {
+            this.repo = repo;
+        }
+
+        public async Task<ReceiptRecoveryContentRuleResult<ReceiptRecoveryContentRules>> GetResult(List<ReceiptRecoveryMovement> movements, Guid notificationId)
+        {
+            var actualMovements = await repo.GetAllMovements(notificationId);
+
+            List<int> shipments = new List<int>();
+            MessageLevel result = MessageLevel.Success;
+
+            foreach (var movement in movements.Where(p => !p.MissingReceivedDate && !p.MissingRecoveredDisposedDate))
+            {
+                var actualMovement = actualMovements.FirstOrDefault(p => p.Number == movement.ShipmentNumber);
+
+                if (actualMovement.Status == MovementStatus.Captured)
+                {
+                    if (actualMovement.Date < DateTime.Now)
+                    {
+                        result = MessageLevel.Error;
+                        shipments.Add(movement.ShipmentNumber.GetValueOrDefault());
+                    }
+                }
+                else if (actualMovement.Status != MovementStatus.Submitted)
```

Comment:
We also need to check for Status of Submitted but with the actual date of shipment to be in the past. So for this case, I think you can just convert the `else if` into an `else` then insert a new `else if` to check for status submitted and ensure that the date is in the past.